bin/node/node/src/chain_spec.rs

```diff
@@ -110,13 +157,19 @@ fn testnet_genesis(
 			balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
 		}),
 		pallet_aura: Some(AuraConfig {
-			authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
+			authorities: Vec::new(),
```

Comment:
It's not necessary, cause afair internally in the runtime there is a callback anyway to initialize genesis keys, right?

Reply:
Yes. And it panics if authorities are set both in session and aura/grandpa config (even if they're the same)